pkg/horizon/deployment.go

```diff
@@ -113,6 +113,22 @@ func Deployment(
 				Spec: corev1.PodSpec{
 					ServiceAccountName: instance.RbacResourceName(),
 					Containers: []corev1.Container{
+						// the first container in a pod is the default selected
+						// by oc log so define the log stream container first.
+						{
+							Name: instance.Name + "-log",
+							Command: []string{
+								"/bin/bash",
+							},
+							Args:  []string{"-c", "tail -n+1 -F " + LogFile},
+							Image: instance.Spec.ContainerImage,
+							SecurityContext: &corev1.SecurityContext{
+								RunAsUser: &runAsUser,
```

Comment:
There's no longer a `runAsUser` variable defined in this file as of:
https://github.com/openstack-k8s-operators/horizon-operator/commit/df907dfeffc6237046c8060a373893654b19ff57#diff-e1c0b69268828ec2f801a92017d9545c992042890e8e8b3a47c3858b73d1b860L57

So this will need to be updated now inline with that effort to remove the root user permissions.

This is causing the failure in `pre-commit` and `kuttl` tests.